### PR TITLE
Refactor/shared model compile time query

### PR DESCRIPTION
**Pull request by Akvinikym**
### Description of the Change

Moved definitions of methods, constructors and some local variables of queries in interfaces and backend protobuf into separate .cpp files from headers. It reduced build time of Iroha - the process was launched via ```make -j 2```:
Before: ```user 3156.97s, system 242.83s```. Total 3 399,8s
After: ```user 3063.83s, system 240.38```. Total 3 304,21s

### Benefits

The build time was reduced by 2.89%

### Possible Drawbacks 

More files to maintain

### CODE REVIEW

File: shared_model/interfaces/queries/impl/get_account_assets.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    std::string GetAccountAssets::toString() const {
+      return detail::PrettyStringBuilder()
+          .init("GetAccountAssets")
+          .append("account_id", accountId())
+          .append("asset_id", assetId())
+          .finalize();
+    }
+
+    bool GetAccountAssets::operator==(const ModelType &rhs) const {
+      return accountId() == rhs.accountId() and assetId() == rhs.accountId();
```
[the remark this answers is not in the document]
Comment (Akvinikym) [reply shared_model/interfaces/queries/impl/get_account_assets.cpp]: Whom should I assign the task to?

File: shared_model/backend/protobuf/queries/impl/proto_get_account.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "backend/protobuf/queries/proto_get_account.hpp"
+#include "utils/lazy_initializer.hpp"
```
Comment (nickaleks) [inline shared_model/backend/protobuf/queries/impl/proto_get_account.cpp]: Unused include, please look at the other proto files as well

File: shared_model/bindings/model_crypto.cpp [hunk 3]
```diff
@@ -30,7 +30,7 @@ namespace shared_model {
         const std::string &private_key) {
       auto byte_string = iroha::hexstringToBytestring(private_key);
       if (not byte_string) {
-        throw std::runtime_error("invalid seed");
+        throw std::invalid_argument("invalid seed");
```
Comment (lebdron) [inline shared_model/bindings/model_crypto.cpp]: Please remove this unrelated change

File: shared_model/interfaces/queries/impl/blocks_query.cpp [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    bool BlocksQuery::operator==(const ModelType &rhs) const {
+      return creatorAccountId() == rhs.creatorAccountId()
+          and queryCounter() == rhs.queryCounter()
+          and createdTime() == rhs.createdTime()
+          and signatures() == rhs.signatures();
+    }
+
+  }  // namespace interface
+}  // namespace shared_model
+
```
Comment (lebdron) [inline shared_model/interfaces/queries/impl/blocks_query.cpp]: This line can also be removed
